plugin/src/main/kotlin/org/jetbrains/research/pynose/plugin/util/UnittestInspectionsUtils.kt

```diff
@@ -0,0 +1,113 @@
+package org.jetbrains.research.pynose.plugin.util
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.util.PsiTreeUtil
+import com.jetbrains.python.psi.*
+import com.jetbrains.python.psi.resolve.PyResolveContext
+import com.jetbrains.python.pyi.PyiFile
+
+class UnittestInspectionsUtils : GeneralInspectionsUtils() {
+    companion object {
```

Comment:
Maybe it would be better to create such utils as extensions to python's psi classes. Anyway, instead of using `companion object` here, you can just declare a global `object`